src/documentation/0023-update-npm-dependencies/index.md

```diff
@@ -44,41 +44,41 @@ and this is an extract of `package-lock.json`, where we removed the nested depen
 }
 ```
 
-Now those 2 files tell us that we installed version `1.3.1` of cowsay, and our rule for updates is `^1.3.1`, which for the npm versioning rules means that npm can update to patch and minor releases: `1.3.2`, `1.4.0` and so on.
+Now those 2 files tell us that we installed version `1.3.1` of cowsay, and our [npm versioning rule](https://docs.npmjs.com/about-semantic-versioning) for updates is `^1.3.1`. This means npm can update to patch and minor releases: `1.3.2`, `1.4.0` and so on.
```

Comment:
Just wondering if it's worth linking to the nodejs.dev semantic versioning page: https://nodejs.dev/learn/semantic-versioning-using-npm

Reply:
Didn't know how common that was in your docs, to cross-link anyway. I figured if a brief explanation was being attempted, the user may desire more information.